testing/go/create_view_test.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2024 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package _go
+
+import (
+	"testing"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+func TestCreateViewStatements(t *testing.T) {
+	RunScripts(t, createViewStmts)
+}
+
+var createViewStmts = []ScriptTest{
+	{
+		Name: "basic create view statements",
+		SetUpScript: []string{
+			"create table t1 (pk int);",
+			"insert into t1 values (1), (2), (3), (1);",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "create view v as select * from t1 order by pk;",
+				Expected: []sql.Row{},
+			},
+			{
+				Query:    "select * from v;",
+				Expected: []sql.Row{{1}, {1}, {2}, {3}},
+			},
+		},
+	},
+	{
+		Name: "create view from view",
+		SetUpScript: []string{
+			"create table t1 (pk int);",
+			"insert into t1 values (1), (2), (3), (1);",
+			"create view v as select * from t1 where pk > 1;",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "create view v1 as select * from v order by pk;",
+				Expected: []sql.Row{},
+			},
+			{
+				Query:    "select * from v1;",
+				Expected: []sql.Row{{2}, {3}},
+			},
+		},
+	},
+	{
+		Name: "view with expression name",
```

Comment:
Name is a little confusing. Is this the `name` not supposed to be there? Or is the name supposed to be an expression?